### PR TITLE
Added region for currency. 

### PR DESCRIPTION
**Pull request by rkugot**
Add region controller

### CODE REVIEW

**Comment by misteral** [comment]
@artem-russkikh Please say me what u think about add that into multy currency gem?

File: app/controllers/spree/region_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      if current_order.present?
+        # Make sure that we update the current order, so the currency change is reflected.
+        current_order.update_attributes!(currency: @currency.iso_code)
+        # Make sure that we update taxes
+        # https://github.com/spree/spree/blob/bc0c3679fb137394c59cfe5651ffe561ee2e4722/core/app/models/spree/order.rb#L285
+        current_order.create_tax_charge!
+        current_order.update!
+      end
+
+      if Spree::Config[:allow_currency_change]
```
Comment (artem-russkikh) [inline app/controllers/spree/region_controller.rb]: I think `if` should wrap all logic, including order update

**Review by artem-russkikh — CHANGES_REQUESTED** [review]

File: app/controllers/spree/region_controller.rb (hunk 1 above)
Comment (misteral) [reply app/controllers/spree/region_controller.rb]: @rkugot please fix

Comment (rkugot) [reply app/controllers/spree/region_controller.rb]: @artem-russkikh but order update already in `if current_order.present?`

Comment (artem-russkikh) [reply app/controllers/spree/region_controller.rb]: @rkugot if we toggle off checkbox Spree::Config[:allow_currency_change] from admin area, we don't need at all functional with updating currency of `current_order`, yes, `current_order.present?` is already checked, but it's another logic (presence of order, not currencies logic).

Comment (rkugot) [reply app/controllers/spree/region_controller.rb]: @artem-russkikh please check last commit

**Review by artem-russkikh — APPROVED** [review]
Thank you, looks good!

**Comment by artem-russkikh** [comment]
@misteral I think now this PR can be merged

**Comment by misteral** [comment]
@rkugot please change in casasola

**Pull request merged**